cert/index.md

```diff
@@ -0,0 +1,103 @@
+# gNSI cert Service Protobuf Definition
+**Contributors**: hines@google.com, morrowc@google.com, tmadejski@google.com
+**Last Updated**: 2022-07-12
+
+## Background
```

Comment:
Would this be the right place to also introduce the concept behind Entity ID that allows the gNSI server to manage more than one set of credentials on the host?  Most will think of X.509 being issued to a host and might therefore get confused when first looking into the example flows in the .proto (note labels "1A", "1B", ...etc).

Reply:
If you have words here that'd be fine by me.
I banged away 'tousand monkeys' style in order to get this far :)